mmengine/visualization/vis_backend.py

```diff
@@ -610,3 +610,135 @@ def close(self):
         """close an opened tensorboard object."""
         if hasattr(self, '_tensorboard'):
             self._tensorboard.close()
+
+
+@VISBACKENDS.register_module()
+class MLFlowVisBackend(BaseVisBackend):
+    """MLFlow visualization backend class.
+
+    It can write images, config, scalars, etc. to a
+    mlflow file.
+
+    Examples:
+        >>> from mmengine.visualization import MLFlowVisBackend
+        >>> import numpy as np
+        >>> vis_backend = MLFlowVisBackend(save_dir='temp_dir')
+        >>> img = np.random.randint(0, 256, size=(10, 10, 3))
+        >>> vis_backend.add_image('img', img)
+        >>> vis_backend.add_scaler('mAP', 0.6)
+        >>> vis_backend.add_scalars({'loss': 0.1,'acc':0.8})
+        >>> cfg = Config(dict(a=1, b=dict(b1=[0, 1])))
+        >>> vis_backend.add_config(cfg)
+
+    Args:
+        save_dir (str): The root directory to save the files
+            produced by the backend.
+        exp_name (str, optional): The experiment name. Default to None.
+        tags (dict, optional): The tags to be added to the experiment.
+            Default to None.
+        params (dict, optional): The params to be added to the experiment.
+            Default to None.
+        log_model (bool, optional): Whether to log the model. Default to True.
+    """
+
+    def __init__(self,
+                 save_dir: str,
+                 exp_name: Optional[str] = None,
+                 tags: Optional[dict] = None,
+                 params: Optional[dict] = None,
+                 log_model: bool = True):
+        super().__init__(save_dir)
+        self._exp_name = exp_name
+        self._tags = tags
+        self._params = params
+        self._log_model = log_model
+
+    def _init_env(self):
+        """Setup env for MLFlow."""
+        if not os.path.exists(self._save_dir):
+            os.makedirs(self._save_dir, exist_ok=True)  # type: ignore
+
+        try:
+            import mlflow
+        except ImportError:
+            raise ImportError('Please install mlflow to use '
+                              'MLFlowVisBackend.')  # type: ignore
+        self._mlflow = mlflow
+
+        if self._exp_name is not None:
+            self._mlflow.set_experiment(self._exp_name)
+        if self._tags is not None:
+            self._mlflow.set_tags(self._tags)
+        if self._params is not None:
+            self._mlflow.log_params(self._params)
+
+    @property  # type: ignore
+    @force_init_env
+    def experiment(self):
+        """Return MLFlow object."""
+        return self._mlflow
+
+    @force_init_env
+    def add_config(self, config: Config, **kwargs) -> None:
+        """Record the config to mlflow.
+
+        Args:
+            config (Config): The Config object
+        """
+        return super().add_config(config, **kwargs)
```

Comment:
How about using `log_params` to record the config?

Reply:
I will check if this is possible to add.